source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -38,10 +39,10 @@
      - ✓
      - ✓
      - ✓
-
```

Comment:
Comment: Any reason the space in between table rows was removed? Doesn't seem to cause any issues, so not a blocker. Just mentioning this in case this was not intentional.